Shared/Samples/Run valve isolation trace/README.metadata.json

```diff
@@ -0,0 +1,60 @@
+{
+    "category": "Utility network",
```

Comment:
Can I get you to change this "Utility Networks" to match the [new sample categories](https://devtopia.esri.com/runtime/common-samples/blob/main/designs/categories.md)?